en.md

```diff
@@ -35,6 +35,7 @@ On the Internet, there are many tutorials/courses to learn development, except t
 
 ### HTML/CSS
 
+* [Complete Guide](https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web)
```

Comment:
You can add this link in the french version as well

Reply:
Je parle d'ajouter le lien EN [ici](https://github.com/learndev-info/awesome-learning-dev-fr/pull/21/commits/c906daec27fcd7faa2a382fde38f9b52e3bbb6c8#diff-92fd2d91409a20eff87d8cd801cf4697R49) et pareil pour le JS [ici](https://github.com/learndev-info/awesome-learning-dev-fr/pull/21/commits/c906daec27fcd7faa2a382fde38f9b52e3bbb6c8#diff-92fd2d91409a20eff87d8cd801cf4697R62)

Reply:
Mais à quoi ça sert, il est en anglais sur la page Anglais, et en français sur la page Francais, c'est les mêmes exactement

Reply:
Car un français n'aura pas forcément de mal à suivre un cours en anglais mais n'ira pas forcément sur la page anglaise et car c'est comme ça pour tout les autres truc

Reply:
Bah il lirait la page en français que j'ai mis du coup ? Qui est tout aussi complète ? Je comprends pas

Reply:
Les traductions française de MDN sont dès fois terribles/mauvaises le mieux même pour un français c'est d'utiliser la version anglaise

Reply:
Pour la référence oui, mais pour les guides que j'ai mis en revanche c'est traduit à 99% (ya que genre pour le cours JS la toute fin je crois, et bah c'est simplement en Anglais donc osef).

Il n'y aucune raison que le gars prenne le cours en anglais à la place du cours en français si il est sur la page française, si il préfère l'anglais (je pense pas que ça se puisse mais pourquoi pas) il serait sur la page anglaise. 

La catégorie Anglais de la page française je voyais ça comme les tutos anglais qui n'existent pas en français, hors là c'est pas le cas, pourquoi mettre 2 fois le même tuto, alors qu'en plus il n'y a aucun raison de prendre celui en anglais ?